prng.h

```diff
@@ -40,14 +41,13 @@
  */
 static inline int
 dtls_prng(unsigned char *buf, size_t len) {
-  while (len--)
-    *buf++ = rand() & 0xFF;
+  getrandom(buf, len, 0);
   return 1;
 }
 
 static inline void
 dtls_prng_init(unsigned short seed) {
-	srand(seed);
+  // nothing to do on linux
```

Comment:
When you do this you should add `(void) seed` to avoid warnings caused by unused arguments.